slack_notifier/slack_notifier.py

```diff
@@ -33,6 +33,10 @@ def send_markdown(self: Self, message: str) -> None:
             logger.info('not sending Slack message because message is empty')
             return
 
+        if len(message) > 3000:
```

Comment:
Is there a need to raise error or should it truncate and append a message that it was truncated?

Reply:
@dcrough-ownum currently it is failing silently so this change will ensure the workflow fails and give us a proper error message (it was just `400 invalid_blocks` before). We do not believe this error will occur regularly so its not a high priority to fix right now, but we do need to know when it happens. As far as fixing it, there are multiple options including splitting each message into a separate "block" here which I think would be preferable compared to truncating.